main.cpp

```diff
@@ -1,6 +1,10 @@
 #include <iostream>
+#include "ScreenCapture.h"
+
+int main(int argc, char **argv) {
+    OutputCapture::ScreenCapture *capturer = OutputCapture::ScreenCapture::GetOrCreateInstance();
```

Comment:
what's happened if your GetOrCreateInstance return nullptr because you don't have enough memory ?

Reply:
Same as last comment, yup you're right, but it's just for the demo